### DISCUSSION TITLE
[v2] unable to create admin user

### DISCUSSION THREAD

**Discussion by AminulBD** [medusajs/medusa#7106] [Q&A] [answered]
Hi,

I'm trying to explore v2 of medusa. I've created new project with `--v2` flag. For example:  `npx create-medusa-app@latest --v2`

It's create new project, migrate tables and open url on the browser. But it doesn't create any users. When i try to create user manually it's throw this error `AwilixResolutionError: Could not resolve 'userService'.` with this command ` medusa user -e admin@medusa.test -p secret`

Is there anyone else facing the same issue? Or just me?

**Comment by olivermrbl** [comment, marked as answer]
I appreciate the interest in V2. However, it is not ready for external development yet, so for the time being, this is a wont-fix.

We expect to announce a pre-release/canary within a month.
